src/App.js

```diff
@@ -60,11 +60,11 @@ export class App extends Component {
       ],
       infoText: [
         "",
-        "You see an unusually long and heavy sword lying next to you.",
-        "As you pick it up, you notice a wounded boar charging at you from the corner of your eye.",
+        "You see an unusually long and heavy sword lying next to you. How convenient!",
+        "As you pick it up, you notice a wounded boar charging at you from the corner of your eye. You probably should consider doing something.",
         "After defeating the boar, you spot a hidden cave nearby that feels safe.",
         "You find refuge in the cave.",
-        "On waking up your body is still aching all over. You find a note thanking you for defeating the boar, along with a nice round coin.",
+        "On waking up your body is still aching all over. While wondering how you managed to make it alive, you find a note thanking you for defeating the boar along with a nice round coin.",
```

Comment:
Yeah, not sure about the tone here. It feels a bit out of place with the theme and the feel I'm trying to get at with the UI.

Reply:
Fine by me. I'm reverting.